RELEASENOTES.md

```diff
@@ -1 +1,2 @@
-- Create shared access signature (SAS) with read-only permissions - not read-write - when generating Azure Blob Storage URL
\ No newline at end of file
+- Create shared access signature (SAS) with read-only permissions - not read-write - when generating Azure Blob Storage URL
+- Add support of AWS session token and region when using AWS S3 for archive upload in `gh gei migrate-repo` and `gh bbs2gh migrate-repo`. AWS region can also be passed into `gh gei generate-script` and `gh bbs2gh generate-script` commands.
```

Comment:
```suggestion
- Add support for authenticating with AWS session tokens when using AWS S3 for archive upload in `gh gei` and `gh bbs2gh`. When specifying a session, the AWS region must also be specified.
```

Reply:
adressed in 98de164b93e6c5ad0a9f24c52673cd80b4f9a7fc